roles/contrail_node/tasks/main.yaml

```diff
@@ -0,0 +1,47 @@
+---
+- when:
+  - docker_registry is not defined
```

Comment:
What is the purpose of this when statement?  What docker_registry?  Where would it be defined?  Can you provide some insight into this?